examples/gomoku/gomoku_examples.ipynb

```diff
@@ -0,0 +1,689 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "----------------------------\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
The main logic is a little too complex, especially for a simple gomoku game. It's suggested to simplify the code in main logic.